conf/kaust.config

```diff
@@ -0,0 +1,71 @@
+// KAUST Config Profile
+params {
+    config_profile_name = 'KAUST'
+    config_profile_description = 'Profile for use on King Abdullah Univesity of Science and Technology (KAUST) Ibex Cluster.'
+    config_profile_contact = 'Husen Umer (@kaust.edu.sa)'
+    config_profile_url = 'https://docs.hpc.kaust.edu.sa/quickstart/ibex.html'
+    save_reference = false
+    igenomes_ignore = true
+}
+
+// Load genome resources and assets hosted by the Bioinformatics team on IBEX cluster
+includeConfig '/biocorelab/BIX/resources/configs/genomes.yaml'
+
+singularity {
+  enabled = true
+  autoMounts = true
+  pullTimeout = '60 min'
+  // Use existing images from the centralized library, if available
+  libraryDir = "/biocorelab/BIX/resources/singularity/images/"
+  // Download images that are missing from the library to user space
+  cacheDir = "/home/$USER/.singularity/nf_images/"
+}
+
+process {
+    executor = 'slurm'
+    clusterOptions = "-p batch"
+    maxRetries = 5
+    errorStrategy = { task.exitStatus in [143,137,104,134,139,151,140,247,12] ? 'retry' : 'finish' }
+    beforeScript = 'module load singularity'
+    // Max allowed resources per process on Ibex
+    resourceLimits = [
+        memory: 1600.GB,
+        cpus: 200,
+        time: 10.d
+    ]
+}
+
+// Keep work directory after a successful run
+cleanup = false
```

Comment:
Be wary of this, if users don't clean up their work directories manually and regularly, you may have very filled up hard drives very quickly 

Nf-core pipelines come with a debug profile that can turn this off if necessary

Reply:
thanks for the note, I have now removed this flag. I thought to keep the decision to the user!

Reply:
It's better to keep it,= true actually,

Default is set to  false!